### DISCUSSION TITLE
Does the unavailability of a service justifies receiving an alert?

### DISCUSSION THREAD

**Discussion by cpipilas** [netdata/netdata#12197] [Ideas]
We all know how important is for monitoring the availability of your services. But we also know that getting an alert each time a service goes down may not be that helpful because one may intentionally stopped that service and this is not actually an availability issue anymore, right?

So let's say you run an online store and during a maintenance, someone accidentally stops MySQL. A lot of services that depend on the database will be affected, failures will be reported, alerts will be raised. All good but the root cause, the fact that MySQL server has been stopped, will be still hidden or at least not clearly visible.

Properly identifying the root cause of such issues, with proper alerting and providing a root cause analysis would be really helpful for everyone. 

What you say, any further ideas/comments?

**Comment by ilyam8** [comment]
> someone accidentally stops MySQL

[meatboy.jpg](https://static.wikia.nocookie.net/supermeatboy/images/d/d4/Meatysticker2.png/revision/latest?cb=20121208152014) 😄 

---

> failures will be reported, alerts will be raised
> the root cause will be still hidden or at least not clearly visible

Are they really mutually exclusive things?

**Reply by cpipilas** [reply]
They are not. But knowing the root cause of a number of alerts may help to actually reduce their number/frequency and emphasise only in the root cause of the problem (e.g. all these 15 alerts are due to a service availability issue), which at the end of the day is what any user would like to know.
